src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -441,20 +456,20 @@ export default {
 // Standalone styling, independent from server
 .rich-contenteditable__input {
 	overflow-y: auto;
-	width: auto;
+	width: 100%;
 	margin: 0;
-	padding: 6px;
+	padding: 8px;
 	cursor: text;
 	white-space: pre-wrap;
 	word-break: break-word;
 	color: var(--color-main-text);
 	border: 1px solid var(--color-border-dark);
-	border-radius: var(--border-radius);
+	border-radius: var(--border-radius-large);
 	outline: none;
 	background-color: var(--color-main-background);
 	font-family: var(--font-face);
 	font-size: inherit;
-	min-height: $clickable-area;
+	min-height: var(--default-line-height);
```

Comment:
With this the default looks off and the height is 37px.
![Capture d’écran_2020-11-13_09-21-42](https://user-images.githubusercontent.com/14975046/99045465-a8f44b00-2591-11eb-99c5-78f9b4d0ca05.png)